coalib/testing/BaseTestHelper.py

```diff
@@ -0,0 +1,36 @@
+from coala_utils.Comparable import Comparable
+
+
+class BaseTestHelper:
+    """
+    This is a base class for all Bears' tests of coala's testing API.
+    """
+
+    def assertComparableObjectsEqual(self, observed_result, expected_result):
+        """
+        Compares individual fields of the Result object and yields messages in
+        case of an attribute mismatch.
+        """
+        if len(observed_result) == len(expected_result):
+            messages = ''
+            for observed, expected in zip(observed_result, expected_result):
+                if (isinstance(observed, Comparable)
+                        and isinstance(expected, Comparable)) \
```

Comment:
I might be wrong here but line breaking using `\` is not recommended.

Reply:
That gives syntax error